docs/blog/2019-02-20-getting-started-with-gatsby-themes/index.md

```diff
@@ -0,0 +1,299 @@
+
+```javascript
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-mdx`,
+      options: {
+        defaultLayouts: {
+          default: require.resolve("./src/components/layout.js"),
+        },
+      },
+    },
+    {
+      resolve: `gatsby-plugin-page-creator`,
+      options: {
+        path: `${__dirname}/src/pages`,
+      },
+    },
+  ],
+}
+```
+
+Now, to make sure _site_ is linked to _theme_.
+
+```
+yarn
+yarn workspaces info
+```
+
+Your workspace info should look similar to this:
+
+```shell
+{
+  "site": {
+    "location": "site",
+    "workspaceDependencies": ["theme"],
+    "mismatchedWorkspaceDependencies": []
+  },
+  "theme": {
+    "location": "packages/theme",
+    "workspaceDependencies": [],
+    "mismatchedWorkspaceDependencies": []
+  }
+}
+```
+
+Lastly, you're going to want to add a _gatsby-config.js_ file to your _site_ directory.
+
+```javascript
+module.exports = {
+  __experimentalThemes: ["theme"],
+}
+```
+
+While you're still in the _site_ directory, you are going to create an _index.mdx_ file in the pages folder.
+
+`site/src/pages/index.mdx`
+
+Your website content goes in _index.mdx_.
+
+# Styling Layout and Components
+
+Next, you will navigate to the _theme_ directory. You will then create a _components_ folder under _src_, and in components you create a _layout.js_ file.
+
+`packages/theme/src/components/layout.js`
+
+Inside of your _layout.js_ file, you can add your styling.
+
+```javascript
+export default ({ children }) => (
+  <div
+    style={
+      {
+        //Layout styling //
+      }
+    }
+  >
+    {children}
+  </div>
+)
+```
+
+To make sure your _layout.js_ file is connected to your theme you will navigate to your _gatsby-config.js_ file in your _theme_ directory. You will add defaultLayouts under options and make sure that the _layout.js_ is required.
+
+```javascript
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-mdx`,
+      options: {
+        defaultLayouts: {
+          default: require.resolve("./src/components/layout.js"),
+        },
+      },
+    },
+    {
+      resolve: `gatsby-plugin-page-creator`,
+      options: {
+        path: `${__dirname}/src/pages`,
+      },
+    },
+  ],
+}
+```
+
+If you want to reuse a specific style, you can create styled components. In your components directory, you will create a new file (for example: _header.js_).
+
+Here is an example of how you can set-up your styled component in _header.js_. Please make sure you write css-in-javascript when styling your div.
+
+```javascript
+export default ({ children }) => (
+  <section
+    style={{
+     // Header Specific Styling //
+    }}
+  >
+    {children}
+  </section>
+);
+➜  co
+```
+
+To import your styled components, go to _index.js_
+
+`packages/theme/index.js`
+
+You will then export your component.
+
+`export {default as Header} from './src/components/header'`
```

Comment:
Should we make this a code block with the title?

```
```javascript:title=packages/theme/index.js